ui/components/ui/form-field/form-field.js

```diff
@@ -67,13 +73,35 @@ export default function FormField({
             {tooltipText && (
               <InfoTooltip position="top" contentText={tooltipText} />
             )}
+            {tooltipContentText && (
+              <Box display={DISPLAY.INLINE_BLOCK}>
+                <Tooltip
+                  interactive
+                  position="top"
+                  html={
+                    <Box
+                      margin={3}
+                      className="form-field__heading-title__tooltip"
+                    >
+                      {tooltipContentText}
+                    </Box>
```

Comment:
Okay Yikes. After reading this I am gonna have to send this back to the drawing board in terms of composability and design system. What you've done here is add a *second set of tooltip props that can't be combined with the others* If someone were to provide both tooltipContentText and tooltipText it'd result in TWO icons with tooltips.

@georgewrmarshall can I get your opinion: Should we convert InfoTooltip to 'IconWithTooltip" and use a level "warning | info | error" to change the icon and color?  Or create a new component in the design system that serves this purpose.

Reply:
@brad-decker Yes, I know that this is written ugly and it yikes, but I saw that the property `tooltipText` have only `InfoTooltip`. I must have warning and question icon with tooltip. Base on that, I added this like it is now. I knew it is not the best approach at all. I will wait to @georgewrmarshall answer on this and I will made appropriate change.

Reply:
@VSaric my comment wasn't meant to be derogatory towards your work, I meant that I could see the predicament you are in and we need to think about the solution a bit more to provide you some guidance. I apologize for not being clearer with my intentions in the original comment :(.

Reply:
@georgewrmarshall Can we get your opinion on this?

Reply:
Thank you @georgewrmarshall for answer on this! I added a `custom-spending-cup-tooltip.js` file and imported in `custom-spending-cup.js`. Done [here](https://github.com/MetaMask/metamask-extension/pull/15522/commits/190f93d7a3b816927c6d40f7039080fdc87be55b). Let me know if it is okay.
Thank you in advance! 😄

Reply:
great suggestion @georgewrmarshall and @VSaric great work on the cleanup. It already looks 1000% better. I may not get time to review fully today but if i have time i will. If not, Monday morning i will review fully :)